scripts/Jenkinsfile

```diff
@@ -50,9 +58,24 @@ pipeline
     {
          always 
          {
-             junit 'tests/tests_log.xml'
-             echo 'Cleaning Workspace'
-             deleteDir() 
+            script
+            {
+                try 
+                {
+                    sh '''
+                    docker rm -f ${BRANCH_NAME}
```

Comment:
Since the docker is specific to SDK do we need to docker image as well?

Reply:
A new jenkins pipeline is setup to fetch docker with latest SDK nightly.